File: 241108/윤년 횟수 구하기/number-of-leap-years.py
n = int(input())
cnt_y = 0
for i in range(1, n+1):
    if i % 400 != 0 and i % 100 == 0:
        cnt_y -= 1
    if i % 4 == 0:
        cnt_y += 1
print(cnt_y)

File: 241108/윤년 횟수 구하기/number-of-leap-years.c
#include <stdio.h>

int main() {
    int n, cnt_y = 0;
    scanf("%d", &n);

    for (int i = 1; i <= n; i++){
        if ((i % 4 == 0 && i % 100 != 0) || i % 400 == 0)
            cnt_y++;
    }
    printf("%d", cnt_y);
    return 0;
}